apps/zero-runtime-next-app/src/app/globals.css

```diff
@@ -1,22 +1,26 @@
-* {
-  box-sizing: border-box;
-  padding: 0;
-  margin: 0;
-}
+@layer reset, mui;
 
-html,
-body {
-  max-width: 100vw;
-  overflow-x: hidden;
-}
+@layer reset {
```

Comment:
This is to make the emotion overrides the `globals.css`, not related to zero-runtime.